aep/general/0151/aep.md.j2

```diff
@@ -75,10 +75,23 @@ but is not obligated to do so:
 
 - Resources that accept multiple parallel requests **may** place them in a
   queue rather than work on the requests simultaneously.
-- Resource that does not permit multiple requests in parallel (denying any new
-  request until the one that is in progress finishes) **must** return
-  `409 Conflict` if a user attempts a parallel request, and include an error
-  message explaining the situation.
+
+{% tab proto %}
+
+- A Resource that does not permit multiple requests in parallel (denying any new
+  request until the one that is in progress finishes) **must** return `ABORTED`
+  if a user attempts a parallel request, and include an error message explaining
```

Comment:
We should not encourage clients to parse error messages. All errors should have an error message, there is no need to mention it here.

We can simplify the wording like this:
If a resource doesn't support concurrent mutation requests, the server must return an ABORTED error if the resource has an outstanding mutation.

Reply:
I agree with not parsing the error. The error message guidance really should be in the *should* category anyway - since it won't block clients from consuming you.

Will update the wording based on your suggestion! thanks.